package.json

```diff
@@ -23,13 +23,15 @@
     "git graph"
   ],
   "scripts": {
-    "build:dev": "webpack --mode development --progress --color",
-    "build:prod": "webpack --mode production --progress --color",
-    "build": "concurrently \"yarn build:dev\" \"yarn build:prod\"",
+    "clean": "rimraf dist",
+    "build:code": "node .esbuild/esbuild.cjs",
+    "build:types": "tsc -p ./tsconfig.json --emitDeclarationOnly",
+    "build:webpack": "webpack --mode production --progress --color",
+    "build:watch": "yarn build:code --watch",
+    "build": "yarn clean; concurrently \"yarn build:code\" \"yarn build:types\"",
```

Comment:
@knsv There are changes to the build commands.

- `build:code`: Only builds code. Does not typecheck. (The editor will be verifying types anyway. This is the fastest)
- `build:types`: Only builds types using tsc. Slower than just esbuild.
- `build:webpack`: Uses `eslint-loader` within webpack to build as we were doing previously.
- `build`: Builds both code & types.